trycourier/client.py

```diff
@@ -368,7 +383,8 @@ def create_brand(self,
                      name,
                      settings,
                      id=None,
-                     snippets=None):
+                     snippets=None,
+                     idempotency_key=None):
```

Comment:
Would it make more sense to make this part of an options hash? This syntax makes it feel like it's part of the object itself. I'm also a little curious how it would scale if other "header" options were potentially added in the future.

Reply:
I kind of kicked back and forth how to best do this. I could use `**kwargs`. Does the SDK user need to know or care that it's a header?

Reply:
No, they definitely do not need to know that it's a header. I actually went out and looked at stripe's implementation and they did it just like you did:

```
import stripe
stripe.api_key = "sk_test_4eC39HqLyjWDarjtT1zdp7dc"

charge = stripe.Charge.create(
  amount=2000,
  currency="usd",
  description="My First Test Charge (created for API docs)",
  source="tok_mastercard", # obtained with Stripe.js
  idempotency_key='hDq51qOEtiioAh6Y'
)
```